docs/appendix/troubleshooting.md

```diff
@@ -110,6 +110,22 @@ sudo apt install jackd qjackctl libpulse-dev pulseaudio
 
 This error can be safely ignored if you do not intend to use audio on your machine.
 
+## Common ESP32 viam-micro-server errors
```

Comment:
These errors can both only occur on when doing the build setup on https://docs.viam.com/installation/viam-micro-server-dev/ . Otherwise there is no makefile and you do not call `build-esp32-bin`

Reply:
ok, sounds good!